calliope/test/test_core_model.py

```diff
@@ -34,3 +34,9 @@ def test_info_minimal_model(self):
         model = calliope.Model(model_location)
 
         model.info()
+
```

Comment:
There are future warning tests in `test_core_preprocess` in a class called (now misnamed, perhaps) `TestDeprecationWarnings`. Might be best to keep them all in one place, seeing as much of their addition is a copy-paste job. But then maybe we should just have a new file for Future/Deprecation warning capture?

Reply:
Yeah agreed, that is not ideal. I usually like to have one test module per production module which makes it simple to find the corresponding tests. So if there is a `core/model.py`, there is as well a `tests/core/test_model.py` that contains all tests. That's why I intuitively moved it there.

I like the idea of moving everything into one test module `test_core_future_warnings`.

Reply:
Moving to a `test_core_future_warnings` seems good -- and we do need to better organise tests. Corresponding test module for each production module would sound sensible for when we do that.